### PR TITLE
[DOCS] EQL: Document `stringContains` function

### PR DESCRIPTION
**Pull request by jrodewig**
Documents the EQL `stringContains` function.

Relates to #54380 and #54136

### CODE REVIEW

File: docs/reference/eql/functions.asciidoc [hunk 1]
```diff
@@ -223,6 +224,72 @@ field datatypes:
...
+// process.command_line = [ "start regsvr32.exe", "start explorer.exe" ]
+stringContains(process.command_line, "regsvr32")  // returns false
+stringContains(process.command_line, "explorer")  // returns true
+
+// empty strings
+stringContains("", "")                            // returns false
+stringContains("start regsvr32.exe", "")          // returns false
+
+// null handling
+stringContains(null, "regsvr32")                  // returns null
```
Comment (rw-access) [inline docs/reference/eql/functions.asciidoc:252]: Based off the conversation today, `null` in any required input will lead to a `null` output
```suggestion
stringContains(null, "regsvr32")                  // returns null
stringContains(process.command_line, null)  // returns null
```

Comment (jrodewig) [reply docs/reference/eql/functions.asciidoc:248]: 👍 Updated with 8bcb8f8f4afbc2edc48530396a028acbc4f30e36.

File: docs/reference/eql/functions.asciidoc [hunk 2]
```diff
@@ -482,6 +483,65 @@ If using a field as the argument, this parameter does not support the
...
+
+[%collapsible]
+====
+*Example*
+[source,eql]
+----
+// process.command_line = "start regsvr32.exe"
+stringContains(process.command_line, "regsvr32")  // returns true
+stringContains(process.command_line, "start ")    // returns true
+stringContains(process.command_line, "explorer")  // returns false
```
Comment (astefan) [inline docs/reference/eql/functions.asciidoc:500]: I would add one additional example, to show a less obvious usage: `stringContains(process.path, process.name)`.

**Review by astefan — APPROVED** [review]
LGTM.
Suggested, also, another example to add.

File: docs/reference/eql/functions.asciidoc (hunk 2 above)
Comment (jrodewig) [reply docs/reference/eql/functions.asciidoc:500]: Added with 5ccc1986852. Thanks!

**Comment by jrodewig** [comment]
### Backport commits
master cde5fc1 
7.x 5981412

**Pull request merged**